tensorflow_datasets/core/download/downloader.py

```diff
@@ -182,7 +178,12 @@ def _sync_download(self, url, destination_path):
         int(response.headers.get('Content-length', 0)) // unit_mb)
     with tf.io.gfile.GFile(path, 'wb') as file_:
       checksum = self._checksumer()
-      for block in response.iter_content(chunk_size=io.DEFAULT_BUFFER_SIZE):
+      if not url.startswith("ftp"):
+        iterator = response.iter_content(chunk_size=io.DEFAULT_BUFFER_SIZE)
+      else:
+        iterator = iter(lambda: response.read(10), b'')
```

Comment:
instead of 10, use io.DEFAULT_BUFFER_SIZE

Reply:
Oops, Missed that part. Correcting it.